src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -0,0 +1,517 @@
+    const {
+      firstName,
+      lastName,
+      birthDate,
+      email,
+      ssn,
+      startDate,
+      phoneNumber,
+      dataCollectionId,
+    } = form
+    return createQuoteBundle({
+      variables: {
+        locale: locale.isoLocale,
+        quoteCartId,
+        quotes: getUniqueQuotesFromVariantList(quoteBundleVariants).map(
+          ({ data: { type, typeOfContract } }) => {
+            return {
+              firstName,
+              lastName,
+              email,
+              birthDate,
+              ssn,
+              startDate,
+              phoneNumber,
+              dataCollectionId,
+              data: {
+                ...form.data,
+                type,
+                typeOfContract,
+              },
+            }
+          },
+        ),
+      },
+    })
+  }
+
+  return (
+    <>
+      <OuterWrapper visibilityState={visibilityState}>
+        <ScrollWrapper
+          ref={scrollWrapper as React.MutableRefObject<HTMLDivElement | null>}
+          windowHeight={windowInnerHeight}
+        >
+          <InnerWrapper>
+            <CloseButton onClick={onClose} />
+            <Section>
+              <DiscountTagWrapper>
+                <DiscountTag offerData={offerData} />
+              </DiscountTagWrapper>
+              <Heading>{textKeys.CHECKOUT_HEADING()}</Heading>
+              <PriceBreakdown
+                offerData={offerData}
+                showTotal={true}
+                isLoading={isBundleCreationInProgress}
+              />
+              <CheckoutDetailsForm formikProps={formik} />
+              <StartDateWrapper>
+                <StartDateLabel>
+                  {textKeys.SIDEBAR_STARTDATE_CELL_LABEL()}
+                </StartDateLabel>
+                <StartDate offerData={offerData} refetch={refetch} />
+              </StartDateWrapper>
+              {isUpsellCardVisible && (
+                <UpsellCard
+                  quoteBundleVariants={quoteBundleVariants}
+                  selectedQuoteBundleVariant={selectedQuoteBundleVariant}
+                  onUpsellAccepted={onUpsellAccepted}
+                />
+              )}
+              <InsuranceSummary offerData={offerData} />
+            </Section>
+            <SignDisclaimer
+              privacyPolicyLink={privacyPolicyLink}
+              checkoutMethod={checkoutMethod}
+            />
+          </InnerWrapper>
+          <Sign
+            canInitiateSign={
+              formik.isValid &&
+              !isBundleCreationInProgress &&
+              signUiState !== 'STARTED'
+            }
+            checkoutMethod={checkoutMethod}
+            signUiState={signUiState}
+            onSignStart={handleSignStart}
+          />
+        </ScrollWrapper>
+      </OuterWrapper>
+      <Backdrop visibilityState={visibilityState} onClick={onClose} />
+      <SignFailModal
+        isVisible={isShowingFailModal}
+        onClose={() => setIsShowingFailModal(false)}
```

Comment:
`setIsShowingFailModal` is only called here. So we have broken this functionality